src/dialect/mod.rs

```diff
@@ -304,6 +304,11 @@ pub trait Dialect: Debug + Any {
         false
     }
 
+    /// Returns true if the dialect supports numbers containing underscores
```

Comment:
```suggestion
    /// Returns true if the dialect supports numbers containing underscores. e.g. `10_000_000`
```
Thinking an example would clarify even further what it implies